bindata/manifests/daemon/daemonset.yaml

```diff
@@ -49,9 +49,63 @@ spec:
             - name: bpf-maps
               mountPath: /sys/fs/bpf
               mountPropagation: Bidirectional
+        - name: kube-rbac-proxy
+          image: quay.io/openshift/origin-kube-rbac-proxy:latest
```

Comment:
this is need to be a variable that we can render like DaemonSet image and we can default to quay in env.yaml similar to how we do DS image